lib/model/model.go

```diff
@@ -104,6 +104,9 @@ type Model interface {
 	FolderStatistics() (map[string]stats.FolderStatistics, error)
 	UsageReportingStats(version int, preview bool) map[string]interface{}
 
+	PendingDevices() (map[protocol.DeviceID]db.ObservedDevice, error) //FIXME use string as key, like for DeviceStatistics?
```

Comment:
I like IDs much better than strings. Let the api deal with serialisation.

Reply:
Agreed, leaving it at `protocol.DeviceID` then.